src/backend/distributed/commands/multi_copy.c

```diff
@@ -3187,26 +3190,47 @@ CheckCopyPermissions(CopyStmt *copyStatement)
 	rel = table_openrv(copyStatement->relation,
 	                  is_from ? RowExclusiveLock : AccessShareLock);
 
-	range_table = CreateRangeTable(rel, required_access);
+	range_table = CreateRangeTable(rel);
```

Comment:
@naisila I'm confused now, is this routine `CreateRangeTable` Citus or PG community? It appears to be from the Citus which has signature of ` CreateRangeTable(Relation rel)` ?

Reply:
OK, I think I know what happened, you removed the second parameter in your commits, that explains :)